lib/serverless/pipeline/index.ts

```diff
@@ -1,6 +1,6 @@
 import * as pulumi from "@pulumi/pulumi";
 import * as aws from "@pulumi/aws";
-import { input } from "@pulumi/aws/types/index";
+import { input } from "@pulumi/aws/types";
```

Comment:
Why do we no longer need to point directly to index?

Reply:
I ran an automated import updating command when I moved the directory structure around. This was the result 🤷‍♂

Reply:
But I believe that the two statements are effectively the same.